lib/widgets/layouts/dashboard_layout.dart

```diff
@@ -154,22 +152,27 @@ class DashboardLayoutState extends State<DashboardLayout>
               children: <Widget>[],
             );
         }
-        return Layout(
-          scrollController: widget.scrollController,
-          topNavigation: TopNavigation(
-                  onShowWalletList: () =>
-                      {setState(() => panel.toggle(WalletList()))},
-                  currentScreen: currentRoute(context),
-                  currentWallet: currentWallet)
-              .getNavigationActions(context),
-          dashboardActions: _buildDashboardHeader(),
-          bottomNavigation: isDesktopSize ? null : _buildBottomNavigation(),
-          widgetList: [
-            _body,
-          ],
-          actions: [],
-          slidingPanel: _panelContent,
-        );
+        return BlocBuilder<DashboardBloc, DashboardState>(
+            builder: (BuildContext context, DashboardState state) {
+          Wallet currentWallet =
```

Comment:
Why do we prefer to use the current wallet as a constant within the blocBuilder instead of using a getter?